### ISSUE TITLE
Dice Loss for Data-imbalanced NLP Tasks 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#41]
# TL;DR
NLP classification tasks often use F1 scores for evaluation, but they use cross entropy for optimization, and if using unbalanced data, there exists large gap between them. Therefore, they proposed a loss of DSC that can be interpreted as smoothing out F1 and multiplied by a factor that reduces the coefficient of easily classifiable samples to zero. They confirmed that the loss was effective across a number of models and datasets.
![image](https://user-images.githubusercontent.com/24524018/89900731-15704e80-dc1f-11ea-9d10-d69d1e33ee69.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/1911.02855

# Submission Dates(yyyy/mm/dd)
2019/11/07

# Authors and institutions
Xiaoya Li, Xiaofei Sun, Yuxian Meng, Junjun Liang, Fei Wu, Jiwei Li
- Department of Computer Science and Technology, Zhejiang University 
- Shannon.AI

# Methods

# Results
![image](https://user-images.githubusercontent.com/24524018/89900742-1a350280-dc1f-11ea-8e79-1a36bf9f0c3d.png)

# Comments
ACL2020
